zstacklib/zstacklib/gpu/vendors/nvidia.py

```
@@ -353,6 +354,127 @@ def pre_detach_from_host(cls):
+            return
+
+        current_driver = os.path.basename(os.path.realpath(driver_link))
+        if current_driver != "nvidia":
+            return
+
+        dev_path = cls._get_dev_path(pci_address)
+        if not dev_path:
+            logger.debug("cannot find /dev/nvidia* for pci device %s, skip in-use check", pci_address)
+            return
+
+        r, o, e = bash_roe("fuser %s 2>/dev/null" % dev_path)
```

Ruff (0.15.7)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)